README.md

```diff
@@ -1,3 +1,3 @@
 # Rook Artwork
 
-![Rook Logo](logo/blue.png)
+![Rook Logo](logo/Rook_Logo-black.svg)
```

Comment:
in the main `rook/rook` repo, we scaled the logo down:
https://github.com/rook/rook/pull/3010/files#diff-04c6e90faac2675aa89e2176d2eec7d8

but i'm actually thinking that's not the way to go here in the artwork repo since the logo is literally the only thing in the readme.  i think it looks great at it's 100% size here.